[2025-07-28T13:00:48.216+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T13:00:48.233+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:00:48.237+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:00:48.237+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-28T13:00:48.243+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-27 00:00:00+00:00
[2025-07-28T13:00:48.249+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=185) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T13:00:48.249+0000] {standard_task_runner.py:63} INFO - Started process 187 to run task
[2025-07-28T13:00:48.249+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'carburant_etl_dag', 'transform_data', 'scheduled__2025-07-27T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/etl_carburant.py', '--cfg-path', '/tmp/tmpqejhle8b']
[2025-07-28T13:00:48.250+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask transform_data
[2025-07-28T13:00:48.261+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-28T13:00:48.280+0000] {task_command.py:426} INFO - Running <TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [running]> on host 90fdb4201ee5
[2025-07-28T13:00:48.321+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='carburant_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-27T00:00:00+00:00'
[2025-07-28T13:00:48.322+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T13:00:48.412+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T13:00:48.413+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/transform/transform_data.py", line 9, in transform_xml
    tree = ET.parse(xml_path)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/xml/etree/ElementTree.py", line 1204, in parse
    tree.parse(source, parser)
  File "/usr/local/lib/python3.12/xml/etree/ElementTree.py", line 569, in parse
    self._root = parser._parse_whole(source)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 1, column 2
[2025-07-28T13:00:48.419+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=carburant_etl_dag, task_id=transform_data, run_id=scheduled__2025-07-27T00:00:00+00:00, execution_date=20250727T000000, start_date=20250728T130048, end_date=20250728T130048
[2025-07-28T13:00:48.424+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task transform_data (not well-formed (invalid token): line 1, column 2; 187)
[2025-07-28T13:00:48.431+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-28T13:00:48.442+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-28T13:00:48.443+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-28T13:13:56.423+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T13:13:56.445+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:13:56.450+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:13:56.450+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-28T13:13:56.462+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-27 00:00:00+00:00
[2025-07-28T13:13:56.468+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=189) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T13:13:56.469+0000] {standard_task_runner.py:63} INFO - Started process 193 to run task
[2025-07-28T13:13:56.469+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'carburant_etl_dag', 'transform_data', 'scheduled__2025-07-27T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl_carburant.py', '--cfg-path', '/tmp/tmpx6nx0cii']
[2025-07-28T13:13:56.471+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask transform_data
[2025-07-28T13:13:56.484+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-28T13:13:56.508+0000] {task_command.py:426} INFO - Running <TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [running]> on host 695a2fdba44c
[2025-07-28T13:13:56.567+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='carburant_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-27T00:00:00+00:00'
[2025-07-28T13:13:56.568+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T13:13:56.577+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T13:13:56.793+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/transform/transform_data.py", line 9, in transform_xml
    tree = ET.parse(xml_path)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/xml/etree/ElementTree.py", line 1204, in parse
    tree.parse(source, parser)
  File "/usr/local/lib/python3.12/xml/etree/ElementTree.py", line 569, in parse
    self._root = parser._parse_whole(source)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 1, column 2
[2025-07-28T13:13:56.801+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=carburant_etl_dag, task_id=transform_data, run_id=scheduled__2025-07-27T00:00:00+00:00, execution_date=20250727T000000, start_date=20250728T131356, end_date=20250728T131356
[2025-07-28T13:13:56.806+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 4 for task transform_data (not well-formed (invalid token): line 1, column 2; 193)
[2025-07-28T13:13:56.817+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-07-28T13:13:56.831+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-28T13:13:56.832+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-07-28T13:23:50.355+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-07-28T13:23:50.372+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:23:50.377+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [queued]>
[2025-07-28T13:23:50.377+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-07-28T13:23:50.386+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-27 00:00:00+00:00
[2025-07-28T13:23:50.391+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-07-28T13:23:50.392+0000] {standard_task_runner.py:63} INFO - Started process 192 to run task
[2025-07-28T13:23:50.391+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'carburant_etl_dag', 'transform_data', 'scheduled__2025-07-27T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/etl_carburant.py', '--cfg-path', '/tmp/tmppzpavyr2']
[2025-07-28T13:23:50.394+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask transform_data
[2025-07-28T13:23:50.404+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-28T13:23:50.425+0000] {task_command.py:426} INFO - Running <TaskInstance: carburant_etl_dag.transform_data scheduled__2025-07-27T00:00:00+00:00 [running]> on host 23a33819e131
[2025-07-28T13:23:50.477+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='carburant_etl_dag' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-27T00:00:00+00:00'
[2025-07-28T13:23:50.478+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-07-28T13:23:51.330+0000] {logging_mixin.py:188} INFO - ✅ Données transformées sauvegardées dans /opt/***/data/clean.csv
[2025-07-28T13:23:51.384+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-07-28T13:23:51.385+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-07-28T13:23:51.397+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=carburant_etl_dag, task_id=transform_data, run_id=scheduled__2025-07-27T00:00:00+00:00, execution_date=20250727T000000, start_date=20250728T132350, end_date=20250728T132351
[2025-07-28T13:23:51.439+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-07-28T13:23:51.458+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-28T13:23:51.459+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
